GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2134,6 +2134,16 @@ private void QuickFetch()
             UICommands.RepoChangedNotifier.Notify();
         }
 
+        protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
+        {
+            return !IsDesignMode && HotkeysEnabled
+                && (base.ProcessCmdKey(ref msg, keyData) // upstream
+                    || (!GitExtensionsControl.IsTextEditKey(keyData) // downstream (without keys for quick search)
+                        && (RevisionGridControl.ProcessHotkey(keyData)
+                            || (CommitInfoTabControl.SelectedTab == DiffTabPage && revisionDiff.ProcessHotkey(keyData))
+                            || (CommitInfoTabControl.SelectedTab == TreeTabPage && fileTree.ProcessHotkey(keyData)))));
```

Comment:
Scripts?

Reply:
The hotkeys of scripts are added to the `HotKeys` property of all controls and forms by `HotkeySettingsManager.LoadHotkeys`.